config/defaults.ini

```diff
@@ -416,6 +416,7 @@ enable_p2p_updates = true
 # default_branch (optional) - The default branch to use for installs.
 #   This affects installs during the image build as well as at runtime.
 #   Flatpaks that do not specify a branch will use this by default.
+# prio (optional) - The priority for the remote (see xa.prio).
```

Comment:
Could you add a comment for `noenumerate` here, too? Might also be nice to add a reference to flatpak-remote(5), which you can view online at https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-remote.

Reply:
Okay, done!